packager/util/config.go

```diff
@@ -185,7 +187,11 @@ func ReadConfig(configBytes []byte) (*Config, error) {
 	// TODO(twifkak): Return an error if the TOML includes any fields that aren't part of the Config struct.
 
 	if config.Port == 0 {
-		config.Port = 8080
+		if i, err := strconv.ParseInt(os.Getenv("PORT"), 10, 32); err == nil {
```

Comment:
Can you document this somewhere? (amppkg.example.toml's Port field seems like the logical place.)